exporter/exporterhelper/internal/observability_experimental.go

```diff
@@ -0,0 +1,89 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build enable_unstable
+// +build enable_unstable
+
+package internal
+
+import (
+	"context"
+	"sync"
+
+	"go.opencensus.io/stats"
+	"go.opencensus.io/stats/view"
+	"go.opencensus.io/tag"
+)
+
+var (
+	currentlyDispatchedBatches = stats.Int64(
+		"/currently_dispatched_batches",
+		"Number of batches that are currently being sent",
+		stats.UnitDimensionless)
+
+	totalDispatchedBatches = stats.Int64(
```

Comment:
This seems to add a new metric to unstable. Can we do it in a separate PR to keep the refactoring clearer?